spec/01-unit/04-prefix_handler_spec.lua

```diff
@@ -769,6 +769,15 @@ describe("NGINX conf compiler", function()
       lazy_teardown(function() cleanup() end)
 
       it("injects the wasm{} subsystem", function()
+        finally(function()
+          helpers.unsetenv("KONG_DYNAMIC_MODULES_PATH")
+        end)
+
+        -- set env var for CI
+        local build_root = os.getenv("BUILD_ROOT") or ""
+        local dyn_mod_path = build_root .. "/kong-dev/openresty/nginx/modules"
```

Comment:
Is the build prefix needed here? I _believe_ it would suffice to use a relative path:
```
local dyn_mod_path = "bazel-bin/build/kong-dev/openresty/nginx/modules"
```

Also, we can leave the env var as an override (helpful in local tests):
```
local dyn_mod_path = os.getenv("KONG_DYNAMIC_MODULES_PATH")
if not dyn_mod_path then
  dyn_mod_path = "bazel-bin/build/kong-dev/openresty/nginx/modules"
end
```

then `dyn_mod_path` can be specified in the Kong conf